Polychat2 crash on empty message handle

Polychat server crashes when minecraft server forge client starting up, but there is more instances of empty message handle.
P.S This suppose to be fault for ftbbackup, but after further testing looks like i misunderstood

Versions :

- forge-1.12.2-14.23.5.2860
- minecraft 1.12.2
- polychat-forge1122-2.0.2
- server-2.0.2
- quark latest version

config : 
ftbbackup.cfg
rest default apart token and ids.
```
    # If set to true, no messages will be displayed in chat/status bar.
    B:silent=true
```

Error from polychat2 server

```
Exception in thread "main" java.lang.RuntimeException: Failed to parse/unpack/handle message.%s
        at club.moddedminecraft.polychat.core.messagelibrary.PolychatProtobufMessageDispatcher.handlePolychatMessage(PolychatProtobufMessageDispatcher.java:46)
        at club.moddedminecraft.polychat.server.PolychatServer.spinOnce(PolychatServer.java:124)
        at club.moddedminecraft.polychat.server.PolychatServer.spin(PolychatServer.java:107)
        at club.moddedminecraft.polychat.server.PolychatServer.main(PolychatServer.java:98)
Caused by: java.lang.reflect.InvocationTargetException
        at sun.reflect.GeneratedMethodAccessor11.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at club.moddedminecraft.polychat.core.messagelibrary.PolychatProtobufMessageDispatcher.handlePolychatMessage(PolychatProtobufMessageDispatcher.java:40)
        ... 3 more
Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: 0
        at java.lang.String.substring(String.java:2746)
        at club.moddedminecraft.polychat.server.handlers.protomessages.ChatMessageHandler.sendMessageToDiscord(ChatMessageHandler.java:30)
        at club.moddedminecraft.polychat.server.handlers.protomessages.ChatMessageHandler.handle(ChatMessageHandler.java:23)
        ... 7 more
```


Another observation, seems like when i forcefully close forge cause exactly same issue, forcefully i mean alt+f4 on minecraft server, not verified on linux tho.
Seems somewhere when server finish starting generate empty message at end?

Further research show me that with quark feature you can kill polychat2, by showing item on chat, automatically killed, because start with empty message, latest/debug log is missed.

--- a/server/src/main/java/club/moddedminecraft/polychat/server/handlers/protomessages/ChatMessageHandler.java
+++ b/server/src/main/java/club/moddedminecraft/polychat/server/handlers/protomessages/ChatMessageHandler.java
@@ -26,6 +26,12 @@ public final class ChatMessageHandler {
 
     private void sendMessageToDiscord(ChatProtos.ChatMessage msg) {
         String discordMsg = msg.getMessage().replaceAll("§.", "");
+
+        // Some mods can manage to send messages that are 0 length, so instead of trying to process them and having the server crash
+        // just return
+        if (discordMsg.length() == 0) {
+            return;
+        }
         discordMsg = "`"
                 + discordMsg.substring(0, msg.getMessageOffset()) +
                 "` "

